### DISCUSSION TITLE
Bug: repository duplicated in badges

### DISCUSSION THREAD

**Discussion by jodydonetti** [community/community#27632] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, one of my repo is being counted (and shown) twice in the badges area, once with 16 stars and once with 128 starts.
They are the same repo, the link points to the same location.

See here:

![image](https://user-images.githubusercontent.com/1010086/181807198-047580c1-664a-4663-aab8-207f8007a5f3.png)

Hope this helps.

**Comment by jfullstackdev** [comment, marked as answer]
yup that's normal, 

see this, if you know Linus Torvalds, the creator of Linux OS

![image](https://user-images.githubusercontent.com/47092464/181807535-6efd1864-748c-4f2c-8e46-5b09020287d4.png)

it's 100% unlocked, the pattern is 

16, 128, 512, 4096

so you will get `x3` once that repo stars reach 512

also, it's the same repo, it's the first repo that reached 16 stars in the past, that's why it's selected

**Reply by jodydonetti** [reply]
> if you know Linus Torvalds, the creator of Linux OS

Ahah yeah, I think I've heard of him 😅

Anyway thanks: I knew about the "steps" but I somehow missed that the first was at 16.

Thanks!
